alacritty.yml

```diff
@@ -337,16 +337,16 @@
 # directory of the parent process will be used.
 #working_directory: None
 
-# Windows 10 ConPTY backend (Windows only)
+# WinPTY backend (Windows only)
 #
-# This will enable better color support and may resolve other issues,
-# however this API and its implementation is still young and so is
-# disabled by default, as stability may not be as good as the winpty
-# backend.
+# Alacritty defaults to using the newer ConPTY backend if it is
+# available, since it provides better color support, resolves various
+# other minor rendering bugs and is quite a bit faster. If it is not
+# available, the the WinPTY backend will be used instead.
```

Comment:
I think this might be a bit too much information. Especially things like `resolves various other minor rendering bugs` seems so unspecific, that listing it explicitly will likely just increase the effort it takes to understand this section.

If people really want to know the differences, I'm sure there are better sources, so maybe we can just keep it short like this?

> Alacritty defaults to using the newer ConPTY backend if it is available, since it resolves a lot of bugs and is quite a bit faster. If it is not available, the WinPTY backend will be used instead.